lib/school_code_generator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SchoolCodeGenerator
+  MAX_CODE = 1_000_000
+
+  def self.generate
+    number = Random.new.rand(MAX_CODE)
```

Comment:
Minor: I think you could just use `Kernel#rand` here: https://docs.ruby-lang.org/en/3.2/Kernel.html#method-i-rand

Reply:
Hmm. I started out using `Kernel#rand` but switched to `Random#rand` as I mistakenly thought `Kernel#rand` didn't specify whether the generated value was inclusive of exclusive of the `max` argument. Do you think it's worth switching to `Kernel#rand`?

Reply:
I don't think it matters much